app/api/files/jsRoutes.js

```diff
@@ -51,6 +47,7 @@ export default app => {
       return next();
     },
     validation.validateRequest(saveSchema),
+    // eslint-disable-next-line max-statements
```

Comment:
I tried to reduce the statements in this function first, but it is pretty compact and is old code. I don't think it's worth fixing this at this point.
If you disagree and have an idea on how to achieve it, let me know and I'll be happy to do it! :)

Reply:
No, it's OK.  That is not even part of this development, and this file no longer contains anything you have developed, so no, it's OK.  Let's leave it as is.